pkg/v1/tkg/client/utils.go

```diff
@@ -11,6 +11,10 @@ import (
 	"regexp"
 	"time"
 
+	clusterctlv1 "sigs.k8s.io/cluster-api/cmd/clusterctl/api/v1alpha3"
+
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/clusterclient"
+
```

Comment:
nit: We should organize imports in three separate groups:

1. standard packages (like "regexp" and "time")
2. external packages (like "github.com/imdario/mergo")
3. in-project packages (like "github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/tkgconfighelper")

And sort them (alphanumerically) within each group.

Reply:
@imikushin This was the autogenerated order when I did a `goimports` . Not sure if I can manually update the order without golint complaining